manifests/balancer.pp

```diff
@@ -41,21 +45,27 @@
 define apache::balancer (
   $proxy_set = {},
   $collect_exported = true,
+  $target = undef,
 ) {
   include ::apache::mod::proxy_balancer
 
-  $target = "${::apache::params::confd_dir}/balancer_${name}.conf"
+  if $target {
+    $real_target = $target
```

Comment:
Could you use `$_target = $target` instead of `$real_target = $target` ? Thanks!